docs/guides/transitions.md

```diff
@@ -561,3 +561,26 @@ quietMachine.transition(quietMachine.initialState, { type: 'WHISPER' });
 quietMachine.transition(quietMachine.initialState, { type: 'SOME_EVENT' });
 // => State { value: 'disturbed' }
 ```
+
+## FAQ's
+
+### How do I do if/else logic on transitions?
+
+Sometimes, you'll want to say:
+
+- If _something_ is true, go to this state
+- If _something else_ is true, go to this state
+- Else, go to this state
+
+You can use [guarded transitions](./guards.md#guarded-transitions) to achieve this.
```

Comment:
I think this should also be accompanied by a simple example - just referring to another section seems like a needless overhead for the user. The link to the section is still useful though, if they want to learn more about it etc

Reply:
I actually think a simple example leads to excess duplication/maintenance. I've added PR's to that effect elsewhere. @laurakalbag What do you think?

Reply:
I've added a conceptual example to illustrate the question, but I don't think a concrete example is needed when the link to the explicit docs section is there.

Reply:
I think simple and short examples are useful, especially for folks who are more visual learners and/or aren’t native English speakers.